handbook/brand.md

```diff
@@ -393,6 +393,119 @@ When conducting an incident post-mortem, answer the following three questions:
+The main body of your article.
+
+### Conclusion
+It’s a good idea to finish your article with a clear closing statement.
+
+### Add a customizable CTA
+Add a CTA at the end of your article. See [Customizable CTA](#heading=h.2nnazmktmx29) below for instructions on creating a CTA tailored to your article topic.
+
+## Images & screenshots
+Images are a great way to help engage your readers. But consider the following before including images or screenshots in your article:
+
+- Does the image add value?
+- Is your image likely to go out of date soon? (Consider the long-term maintenance of your article.
+
+## Meta tags
+These tags help … summary of what these are for (todo)
+
+`<meta name="articleTitle" value="Deploying Fleet on Render">
+<meta name="authorFullName" value="Ben Edwards">
+<meta name="authorGitHubUsername" value="edwardsb">
+<meta name="category" value="guides">
+<meta name="publishedOn" value="2021-11-21">
+<meta name="articleImageUrl" value="../website/assets/images/articles/deploying-fleet-on-render-cover-1600x900@2x.jpg">`
+
+## `articleTitle`
+The title of your article. Avoid long titles. As a rule of thumb, your title should not exceed two lines at desktop browser resolution. This is roughly 75 characters (including spaces).
+
+## `authorFullName`
+Add the author’s full name here. Our system does not currently support multiple authors.
+
+## `authorGitHubUsername`
+Add the author’s GitHub username to populate the author’s headshot.
+
+## `category`
+Choose only __one__ of the following categories for your article.
+
+- __Announcements__: company or product announcements (including breaking changes), news, or events.
+- __Engineering__: posts about engineering at Fleet and other engineering-related topics.
+- __Guides__: help articles for using and deploying Fleet.
+- __Podcasts__: podcast-related posts.
+- __Product__: posts related to Fleet features.
+- __Releases__: release posts, security, and patch releases.
+- __Reports__: posts about the industry, data, surveys, etc.
+- __Security__: posts about how we approach security at Fleet and other security-related topics.
+- Success stories: stories from users or customers successfully using Fleet. 
+
+## `publishedOn`
+The date that the article was published. Please follow the correct date format, e.g., __2021-09-29__.
+
+## `articleImageUrl`
+The relative url path for the article cover image. Article images are stored in ../website/assets/images/articles/ See <ins>fleetdm.com image conventions</ins> (todo).
+
+## Customizable CTA
+Use the following code snippet to include an inline CTA (call to action) in your article:
+
+`<call-to-action 
+title=”Give Fleet a try”
+text=”Get up and running with a text environment of Fleet within minutes.”
+primary-button-text=”Get started” 
+primary-button-href=”/get-started?try-it-now” 
+secondary-button-text=”Schedule a demo”
+secondary-button-href=”calendly.com/fleetdm/demo”>
+</call-to-action>`
+___________INSERT_IMAGE_______________
```

Comment:
Having not put an image in Markdown before I could use a little guidance.

Reply:
Ok, let's get together.